### ISSUE TITLE
Limited time format validation

### ISSUE THREAD

**Issue by DavidHenri008** [eclipsesource/jsonforms#1689]
**Describe the bug**
I think the time format validation is too restrictive. It is added in the [validator.ts](https://github.com/eclipsesource/jsonforms/blob/bc819784999c9ebd5fee08421593d2c3995c35b0/packages/core/src/util/validator.ts#L38) and there is no explanation why the default time format is replaced.

**Expected behavior**
I would expect that the default ajv time regex to be used.

**Used Setup (please complete the following information):**
 - Framework: react
 - RendererSet: material

**Comment by DavidHenri008** [comment]
As a note, I was able to circumvent the limitation by doing the following
`
const defaultsAjv = createAjv({ useDefaults: true }).addFormat(
    'time',
    /^(?:[0-2]\d:[0-5]\d:[0-5]\d|23:59:60)(?:\.\d+)?(?:z|[+-]\d\d(?::?\d\d)?)?$/i
  );

return <JsonForms
      data={data}
      schema={schema}
      uischema={uiSchema}
      renderers={materialRenderers}
      cells={materialCells}
      onChange={handleOnChange}
      ajv={defaultsAjv}
    />;
`

**Comment by sdirix** [comment]
Hi @DavidHenri008, good find! Our manual time format is old code when JSON Schema Draft-4 was still new. [It only supported the `date-time` format](https://tools.ietf.org/html/draft-fge-json-schema-validation-00#section-7.3). As we already offered time controls we added our own validator for it. Only starting with [JSON Schema Draft-7 `date` and `time` were added to the spec](https://json-schema.org/draft-07/json-schema-release-notes.html#formats).

Obviously our custom `time` format is no longer needed, so we can actually remove it. Thanks for the report!
